src/partitioning/loadBalance.F90

```diff
@@ -591,32 +593,27 @@ subroutine blockDistribution
     if(commNeglected .and. myID == 0) then
        print "(a)", "#"
        print "(a)", "#                    Warning"
-       print "(a)", "# Communication costs neglected to obtain&
-            & a valid partitioning."
+       print "(a)", "# Communication costs neglected to obtain a valid partitioning."
     endif
 
     ! If the load imbalance tolerance was not met, print a warning
     ! message if I am processor 0.
 
+    loadFormat = "(*(A, F6.3))"
```

Comment:
Any reason to define these here? It does not seem to change, so I would just move this to where its declared. There are other instances of this.

Reply:
Right, thats true. Some of these though feel a bit more like a constant that is just defined once and not set again, a and this is why I mentioned this. Not a big deal and this is fine as is. We can do another round later, defining some of these as `PARAMETER`.